features/FEATURE_BLE/ble/Gap.h

```diff
@@ -736,6 +780,107 @@ class Gap {
         {}
     };
 
+    /**
+     * Privacy Configuration of the peripheral role.
+     *
+     * @note This configuration also apply to the broadcaster role configuration.
+     */
+    struct PeripheralPrivacyConfiguration_t {
+        /**
+         * Indicates if non resolvable random address should be used when the
+         * peripheral advertise non connectable packets.
+         *
+         * Resolvable random address continue to be used for connectable packets.
+         */
+        bool use_non_resolvable_random_address;
+
+        /**
+         * Resolution strategy of initiator resolvable addresses when a
+         * connection request is received.
+         */
+        enum ResolutionStrategy {
+            /**
+             * Do not resolve the address of the initiator and accept the
+             * connection request.
+             */
+            DO_NOT_RESOLVE,
+
+            /**
+             * If a bond is present in the secure database and the address
+             * resolution fail then reject the connection request with the error
+             * code AUTHENTICATION_FAILLURE.
+             */
+            REJECT_NON_RESOLVED_ADDRESS,
+
+            /**
+             * Perform the pairing procedure if the initiator resolvable
+             * address failed the resolution process.
+             */
+            PERFORM_PAIRING_PROCEDURE,
+
+            /**
+             * Perform the authentication procedure if the initiator resolvable
+             * address failed the resolution process.
+             */
+            PERFORM_AUTHENTICATION_PROCEDURE
+        };
+
+        /**
+         * Connection strategy to use when a connection request contains a
+         * private resolvable address.
+         */
+        ResolutionStrategy resolution_strategy;
+    };
+
+    /**
+     * Privacy Configuration of the central role.
+     *
+     * @note This configuration encompass the observer role configuration.
```

Comment:
encompasses
but I also don't understand, do you mean: also used for the observer role configuration?